### ISSUE TITLE
seems error of logits reshape when num_classes != 2

### ISSUE THREAD

**Issue by LilianYe** [HasnainRaz/FC-DenseNet-TensorFlow#2]
In function xentropy_loss() of model.py 
`logits = tf.reshape(logits, [tf.shape(logits)[0], -1, 2])`
should be 
`logits = tf.reshape(logits, [tf.shape(logits)[0], -1, self.num_classes])`
